internal/output/html/vuln_table_template.gohtml

```diff
@@ -15,7 +15,11 @@
       {{ else }}
       <div class="tooltip">
         <div class="clickable" onclick="openVulnInNewTab('{{ $element.ID }}')">{{ $element.ID }}</div>
-        <span class="tooltiptext">Group IDs: {{ join $element.GroupIDs ", " }}</span>
+        <span class="tooltiptext">Group IDs: <br>
```

Comment:
This change also needs to be added to the "uncalled vulnerability" section below. Ideally, we should only have one template, but "called" and "uncalled" vulnerabilities are displayed slightly differently, so I've kept two sections for now. Perhaps we can just merge them.